app/views/includes/foot.jade

```diff
@@ -8,6 +8,8 @@ script(type='text/javascript', src='/lib/underscore/underscore-min.js')
 
 //Bootstrap
 script(type='text/javascript', src='/lib/bootstrap/dist/js/bootstrap.js')
```

Comment:
Hi, @clintfidel seems like you've added another `boostrap.js` on line  22, please get rid of one of them.